testdata/wait-rabbitmq.ps1

```diff
@@ -0,0 +1,21 @@
+[int]$count = 1
+
+Do {
+    $proc_id = (Get-Process -Name erl).Id
+    if (-Not ($proc_id -is [array])) {
+        & 'C:\Program Files\RabbitMQ Server\rabbitmq_server-3.7.3\sbin\rabbitmqctl.bat' wait -t 300000 -P $proc_id
```

Comment:
There is also a hard-coded reference to `cmd /c "C:\Program Files\RabbitMQ Server\rabbitmq_server-3.7.3\sbin\` in appveyor.yml. This includes both installation location and version-specific information. Let's not duplicate this knowledge across scripts. Please export an environment variable with the appropriate path from appveyor.yaml and use the variable instead of hard-coding the path.